website/source/api/index.html.md

```diff
@@ -15,22 +15,35 @@ CRUD operations on nodes, services, checks, configuration, and more.
 ## Authentication
 
 When authentication is enabled, a Consul token should be provided to API
-requests using the `X-Consul-Token` header. This reduces the probability of the
+requests using the `X-Consul-Token` header or with the
+ Bearer scheme in the authorization header. 
+This reduces the probability of the
 token accidentally getting logged or exposed. When using authentication,
-clients should communicate via TLS.
+clients should communicate via TLS. If you don’t provide a token in the request, then the agent default token will be used.
 
-Here is an example using `curl`:
 
-```text
+Below is an example using `curl` with `X-Consul-Token`.
+
+```sh
 $ curl \
     --header "X-Consul-Token: abcd1234" \
```

Comment:
nit: maybe for consistency use `<consul token>` here like the following example?

Reply:
Yea I think thats fine since "abcd1234" isn't meaningful.